src/layouts/docs/index.tsx

```diff
@@ -21,6 +23,12 @@ const DocsLayout: React.FC<DocsLayoutProps> = (props) => (
           {/* <div className={s.versionSwitcher}>VERSION SWITCHER</div> */}
           <main className={s.main} id="main">
             {props.children}
+            {props.githubFileUrl ? (
+              <EditOnGithubLink
+                url={props.githubFileUrl}
+                label="Edit this page on GitHub"
+              />
+            ) : null}
```

Comment:
Thoughts on using `&&` instead of a ternary with a `null` portion?

```suggestion
            {props.githubFileUrl && (
              <EditOnGithubLink
                url={props.githubFileUrl}
                label="Edit this page on GitHub"
              />
            )}
```

Reply:
Thanks for calling this out 💯 This is a holdover from personal habits 😬 have always done this cause I'm paranoid the initial condition will unexpectedly be `0` or `NaN` or something and get rendered (instead of the intended "render nothing"). But with Typescript I guess I can rest a lot easier and enjoy the good live of less ternaries 😌🎉